summoner-cli/src/Summoner/Template/GitHub.hs

```diff
@@ -28,16 +28,22 @@ import Summoner.Settings (Settings (..))
 import Summoner.Text (tconcatMap)
 import Summoner.Tree (TreeFs (..))
 
+import qualified Data.Text as T
 
 gitHubFiles :: Settings -> [TreeFs]
 gitHubFiles Settings{..} =
-    [File ".gitignore" gitignore     | settingsGitHub]
+    [File ".gitignore" (gitignoreDefault <> gitignoreCustom) | settingsGitHub]
  ++ [File ".travis.yml" travisYml    | settingsTravis]
  ++ [File "appveyor.yml" appVeyorYml | settingsAppVeyor]
   where
-    -- create .gitignore template
-    gitignore :: Text
-    gitignore =
+
+    -- additional .gitignore
+    gitignoreCustom :: Text
+    gitignoreCustom = T.intercalate endLine settingsGitignore <> endLine
```

Comment:
I think it would be nice to add a comment like saying `# User specific` (or anything better) just to separate it from the other content. How do you think?